### PR TITLE
Bump continuity

### PR DESCRIPTION
**Pull request by cpuguy83**
Fixes an issue where `CopyDir` would truncate files larger than 2^32
bytes.

https://github.com/containerd/continuity/compare/d8fb8589b0e8e85b8c8bbaa8840226d0dfeb7371...2d3749b4da569ac97ca63dccba5eee4f5ee2beab

### CODE REVIEW

File: vendor/github.com/containerd/continuity/fs/copy.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func copyDirectory(dst, src string, inodes map[uint64]string) error {
 				if err := os.Link(link, target); err != nil {
 					return errors.Wrap(err, "failed to create hard link")
 				}
-			} else if err := copyFile(source, target); err != nil {
+			} else if err := CopyFile(target, source); err != nil {
```
Comment (thaJeztah) [inline vendor/github.com/containerd/continuity/fs/copy.go]: Why was the order of target and source switched?

Comment (thaJeztah) [reply vendor/github.com/containerd/continuity/fs/copy.go]: Ah; https://github.com/containerd/continuity/pull/106#discussion_r179883651

> We usually use LHS notation for the args in this package.

Confusing, as `CopyFileRange()` still uses `src`, `dst`

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Pull request merged**